docs/08-docker/02-windows-docker-images.mdx

```diff
@@ -17,70 +17,94 @@ found in
+Windows 10 SDK is not installed. You can install from here: https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/
+```
 
-### Injecting VC++ into the image
+There are two options to avoid this obstacle:
 
-**To pass VC++ from a host system, two steps are necessary**
+1. (*Recommended*) Inject/mount Microsoft Visual Studio with VC++ and Windows 10 SDK from the host system into the
+   container during runtime.
+2. Build a custom image and push it to a docker registry.
 
-_Add to the system registry (in a container) path and version to Windows SDK_
+#### Option 1. Injecting VC++ from the host system into the container
 
-```powershell
-$windowsSdkDirectory = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Windows\v10.0"
+> These instructions have been validated with Visual Studio 2022 and Unity 2021 LTS.
 
-New-Item -Path $windowsSdkDirectory -Force | Out-Null
+_Install Visual Studio Build Tools_
 
-New-ItemProperty -Path $windowsSdkDirectory `
-     -Name InstallationFolder `
-     -PropertyType String `
-     -Force `
-     -Value "C:\Program Files (x86)\Windows Kits\10\" | Out-Null
+The recommended way to install the correct dependencies on the host is with [Chocolatey](https://chocolatey.org/). This works with Windows 10/11, Windows Server, and Windows Server Core.
 
-New-ItemProperty -Path $windowsSdkDirectory `
-     -Name ProductVersion `
-     -PropertyType String `
-     -Force `
-     -Value $env:WINDOWS_SDK_VERSION | Out-Null
+```powershell
+choco install visualstudio2022-workload-vctools --no-progress -y
 ```
 
-_Mount list of volumes_
+There is a way to install the same dependencies from the graphical [Visual Studio Installer](https://visualstudio.microsoft.com/downloads/), but the workload component options in the installer frequently change.
+
+_The following directories need to be bind-mounted into the container_
 
 - C:\Program Files (x86)\Windows Kits
 - C:\Program Files (x86)\Microsoft Visual Studio
-- C:\Program Files\Microsoft Visual Studio
 - C:\ProgramData\Microsoft\VisualStudio
 
+_Docker run example_
+```powershell
+docker run `
+  -v "C:\Program Files (x86)\Windows Kits:C:\Program Files (x86)\Windows Kits" `
+  -v "C:\Program Files (x86)\Microsoft Visual Studio:C:\Program Files (x86)\Microsoft Visual Studio" `
+  -v "C:\ProgramData\Microsoft\VisualStudio:C:\ProgramData\Microsoft\VisualStudio" `
+  unityci/editor:windows-$env:UNITY_VERSION-windows-il2cpp-$env:UNITY_IMAGE_VERSION
+```
+
 _Gitlab runner config.toml example_
 
 ```toml
 #...
 volumes = [
   "C:\\Program Files (x86)\\Windows Kits:C:\\Program Files (x86)\\Windows Kits",
   "C:\\Program Files (x86)\\Microsoft Visual Studio:C:\\Program Files (x86)\\Microsoft Visual Studio",
-  "C:\\Program Files\\Microsoft Visual Studio:C:\\Program Files\\Microsoft Visual Studio",
   "C:\\ProgramData\\Microsoft\\VisualStudio:C:\\ProgramData\\Microsoft\\VisualStudio"
 ]
 #...
 ```
 
+#### Option 2. Building a custom image
```

Comment:
```suggestion
### Option 2. Building a custom image
```

Reply:
level 4 is correct here. It matches Option 1 and is one depth more than the section it's contained in.